projects/lights_control/src/lights_blinker.c

```diff
@@ -7,6 +7,21 @@
 #include "lights_blinker.h"
 #include "lights_events.h"
 
+// Raises a sync event when the counter reaches sync_count.
+static void prv_sync_counter(LightsBlinker *blinker) {
```

Comment:
This function name is confusing. It sounds like you're synchronizing the counter.